go.mod

```diff
@@ -59,6 +58,11 @@ require (
 	sigs.k8s.io/yaml v1.3.0
 )
 
+require (
+	github.com/jba/templatecheck v0.6.0
+	go.mongodb.org/atlas-sdk/v20230201002 v20230201002.0.0
+)
```

Comment:
any reason to group these separately? can we move them to the general group?

Reply:
Very good point
I did that a couple times.  go mod tidy reverts it. I will investigate further.